edward/stats/distributions.py

```diff
@@ -9,68 +9,139 @@ class Distribution:
     """Template for all distributions."""
     def rvs(self, size=1):
         """
+        Parameters
+        ----------
+        size : int, list, or tuple, optional
+            Number of samples, in a particular shape if specified in a
+            list or tuple with more than one element.
+
+        params : float, np.array, or tf.Tensor
+
         Returns
         -------
         np.ndarray
-            size-dimensional vector; scalar if size=1
+            np.array of dimension (size x shape), where shape is the
+            shape of its parameter argument. For multivariate
+            distributions, shape corresponds to one of the parameter
```

Comment:
``shape`` corresponds to the dimension of the multivariate random variable, e.g., the dimension of ``alpha`` in a ``Dirichlet``, the dimension of ``p`` in a ``Multinomial``, or the dimension of ``mean`` in ``Multivariate_Normal``